R/utils-approved.R

```diff
@@ -0,0 +1,53 @@
+#' Build approved packages and functions tibble
+#'
+#' A utility function to help you build your approve package and functions list.
+#' This can be used by timber to log unnaproved use.
+#'
+#' #' For more details see the help vignette:
+#' \code{vignette("approved", package = "timber")}
```

Comment:
Did not know you could do this...very cool!!  I always just annoyingly hardcore the html